templates/parameter_table.html

```diff
@@ -1,12 +1,7 @@
 <!-- Parameter/arguments table -->
-<div class="eagleTableDisplay" id="parameterTable" tabindex="-1" role="dialog" aria-labelledby="parameterTableTitle">
+<div class="eagleTableDisplay parameterTable" tabindex="-1" role="dialog" aria-labelledby="parameterTableTitle">
```

Comment:
I think we could probably rename this file to be "node_parameter_table.html".

Then we'd have "node parameter table" and "config parameter table" as specific instances of the generic "parameter table".

We'd not only rename the file, but a lot of the classes within.

But renaming everything is a lot of work, so maybe just create another ticket to talk about it/do it later.

Reply:
ive created another ticked for this